app/the_bill.py

```diff
@@ -0,0 +1 @@
+import json
```

Comment:
do you need this?

Reply:
no sorry it's not working task with with unnecessary file.